src/routes/(app)/pos/touch/+page.svelte

```diff
@@ -147,17 +196,35 @@
 </div>
 
 <div class="grid grid-cols-3 gap-4 mt-2">
-	<div class="touchScreen-ticket-menu text-3xl p-4 text-center">TICKETS</div>
+	<button
+		class="touchScreen-ticket-menu text-3xl p-4 text-center"
+		on:click={() => alert('Not developped yet')}>TICKETS</button
+	>
 	<div class="col-span-2 grid grid-cols-3 gap-4">
-		<div class="col-span-1 touchScreen-action-secondaryCTA text-3xl p-4">SAUVER</div>
-		<div class="col-span-1 touchScreen-action-secondaryCTA text-3xl p-4">POOL</div>
-		<div class="col-span-1 touchScreen-action-secondaryCTA text-3xl p-4">OUVRIR TIROIR</div>
+		<button
+			class="col-span-1 touchScreen-action-secondaryCTA text-3xl p-4"
+			on:click={() => alert('Not developped yet')}>SAUVER</button
+		>
+		<button
+			class="col-span-1 touchScreen-action-secondaryCTA text-3xl p-4"
+			on:click={() => alert('Not developped yet')}>POOL</button
+		>
+		<button
+			class="col-span-1 touchScreen-action-secondaryCTA text-3xl p-4"
+			on:click={() => alert('Not developped yet')}>OUVRIR TIROIR</button
+		>
 	</div>
 </div>
 <div class="grid grid-cols-2 gap-4 mt-2">
 	<div class="touchScreen-action-cta text-3xl p-4 text-center">PAYER</div>
 	<div class="grid grid-cols-2 gap-4">
-		<div class="col-span-1 touchScreen-action-cancel text-3xl p-4 text-center">❎</div>
-		<div class="col-span-1 touchScreen-action-delete text-3xl p-4 text-center">🗑️</div>
+		<button
+			class="col-span-1 touchScreen-action-cancel text-3xl p-4 text-center"
+			on:click={() => removeLastItem()}>❎</button
+		>
+		<button
+			class="col-span-1 touchScreen-action-delete text-3xl p-4 text-center"
+			on:click={() => removeAllItems()}>🗑️</button
+		>
```

Comment:
Please use forms instead.

You can set a different `formaction` on each button - and `use:enhance` on the form to make it dynamic.

You can do the `confirm` call in the `use:enhance` body.

You can also make the buttons disabled when there's no items, that way no need to add the check in the callback